cyhy_report/contacts/contacts.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+"""Output all points of contact in the CyHy database in CSV format.
+
+Usage:
+  cyhy-contacts [--section SECTION]
+  cyhy-contacts (-h | --help)
+  cyhy-contacts --version
+
+Options:
+  -h --help                      Show this screen.
+  --version                      Show version.
+  -s SECTION --section=SECTION   Configuration section to use.
+"""
+
+# Standard Python Libraries
+from csv import DictWriter
+import StringIO
+
+# Third-Party Libraries
+from docopt import docopt
+
+# cisagov Libraries
+from cyhy.db import database
+
+def generate_contacts_csv(db):
+    all_request_docs = db.RequestDoc.find().sort("_id", 1)
+    output = StringIO.StringIO()
+
+    fields = (
+        "Org ID",
+        "Org Name",
+        "Org Type",
+        "Org Retired",
+        "Contact Name",
+        "Contact Email",
+        "Contact Type",
+    )
+
+    writer = DictWriter(output, fields)
+    writer.writeheader()
+
+    for doc in all_request_docs:
```

Comment:
Is this doable? It just seems silly to assign the result to a variable and then iterate over it separately. I added a comment just to be as explanatory as the variable name.

```suggestion
    # Iterate through all request documents
    for doc in db.RequestDoc.find().sort("_id", 1):
```